src/plugins/intel_npu/src/plugin/CMakeLists.txt

```diff
@@ -60,11 +61,13 @@ target_include_directories(${TARGET_NAME}
     PRIVATE
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/>
 )
-target_include_directories(${TARGET_NAME}
-    SYSTEM PRIVATE
-        $<TARGET_PROPERTY:LevelZero::LevelZero,INTERFACE_INCLUDE_DIRECTORIES>
-        $<TARGET_PROPERTY:LevelZero::NPUExt,INTERFACE_INCLUDE_DIRECTORIES>
-)
+if(ENABLE_ZEROAPI_BACKEND)
+    target_include_directories(${TARGET_NAME}
```

Comment:
I know this is already existing code, but I wonder if it is really needed. Shouldn't it be included only in zero backend library?

Reply:
yeah, I was also wondering why these are added here. I just didn't want to change the logic about these here.